composer.json

```diff
@@ -16,11 +16,17 @@
     "ext-json": "*",
     "ext-pdo": "*",
     "ext-xsl": "*",
+    "ext-mbstring": "*",
+    "ext-bcmath": "*",
+    "predis/predis": "~0.8",
     "google/apiclient": "~1.1",
     "monolog/monolog": "~1.17",
     "paragonie/random_compat": "~1.2",
     "suin/php-rss-writer": "~1.3",
-    "swiftmailer/swiftmailer": "~5.4"
+    "swiftmailer/swiftmailer": "~5.4",
+    "bernard/bernard": "dev-master#f30c0186f0fb391eb6c7ab28f515bb30d34f402d",
+    "ramsey/uuid": "^3.4",
```

Comment:
Fun fact. 3.2.0 is the latest version of `ramsey/uuid` that works on GAE (to my knowledge).